src/components/member-profile/contribution-type/index.js

```diff
@@ -29,11 +32,19 @@ const ContributionType = (props) => {
   const { fullName, type, imageLink, contributions, devUser, tasks } = props;
 
   const [showMoreContent, setShowMoreContent] = useState(true);
+  const initCount =
+    devUser === true ? INIT_CONTRIBUTIONS : contributions.length;
+  const [count, setCount] = useState(initCount);
 
   const showMoreContentHandler = () => {
+    setCount(initCount);
     setShowMoreContent((prevstate) => !prevstate);
   };
 
+  const loadMoreHandler = () => {
```

Comment:
I think we can refactor this function to just a single line something like
`const loadMoreHandler=()=>setCount(prevCount=>pevCount+INIT_CONTRIBUTIONS)`